src/autorenew-invite/autorenew-invite.component.html

```diff
@@ -0,0 +1,19 @@
+<oui-message data-type="info"
+             data-ng-if="$ctrl.serviceInfos && $ctrl.isAutorenewAllowed && !$ctrl.isInAutorenew() && !$ctrl.isExpired() && $ctrl.shouldBotherWithAutorenew()"
+             data-on-dismissed="$ctrl.stopBotheringWithAutorenew()">
+    <span data-translate="autorenew_alert" data-translate-values="{ product: ('autorenew_alert_' + $ctrl.productType | translate) }"></span>
+    <a class="oui-link"
+       data-ng-if="$ctrl.hasPaymentMean"
+       href="{{:: $ctrl.autorenewUrl }}"
+       data-translate="autorenew_activate">
+    </a>
+    <a href="{{:: $ctrl.guideUrl }}"
```

Comment:
`href` is a bit of a particular case: https://docs.angularjs.org/api/ng/directive/ngHref

```suggestion
     <a data-ng-href="{{:: $ctrl.guideUrl }}"
```